<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Loading_Screen</class>
 <widget class="QMainWindow" name="Loading_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 40px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="Loading_Frame">
      <property name="styleSheet">
       <string notr="true">QFrame {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.9945, y2:1, stop:0.125 rgba(129, 48, 142, 255), stop:1 rgba(63, 31, 95, 255));
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Slogan_Pylearn">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>221</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
background: none;
color:  rgb(255, 170, 255);
font: bold;
font-size: 40px;
}</string>
       </property>
       <property name="text">
        <string>PyLearn</string>
       </property>
      </widget>
      <widget class="QLabel" name="Slogan">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>60</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
background: none;
color: rgb(213, 41, 255);
font-size: 20px;
}</string>
       </property>
       <property name="text">
        <string>ỨNG DỤNG HỌC TẬP</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>160</y>
         <width>511</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar { 	
	border-radius: 20px;
 	color: rgb(255, 170, 255);
 	font: bold;
	font-size: 20px;
	text-align: center;
	background-color: rgba(154, 57, 171, 100);
}

QProgressBar::chunk {
	border-radius: 20px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.551, x2:1, y2:0.579, stop:0 rgba(210, 93, 255, 255), stop:0.642045 rgba(142, 32, 201, 255));
}</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="Loading_label">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>200</y>
         <width>351</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
background: none;
color: rgb(170, 85, 255);
font-size: 18px;
}</string>
       </property>
       <property name="text">
        <string>đang tải ...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_credits">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>290</y>
         <width>651</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: none;
color: rgb(98, 114, 164);</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;Created&lt;/strong&gt;: Tien Huynh @Tok</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
